### ISSUE TITLE
Artist tags not recognized for FLACs uploaded with telegram-upload

### ISSUE THREAD

**Issue by awsms** [Nekmo/telegram-upload#101]
* telegram-upload version: 0.4.0
* Python version: 3.6.9
* Operating System: Ubuntu 18.04
* Dependencies list (run `pip freeze`): absl-py==0.11.0
acme==1.13.0
aigpy==2021.3.1.0
aiohttp==3.7.4.post0
alabaster==0.7.12
ansible==3.0.0
ansible-base==2.10.6
appdirs==1.4.4
argcomplete==1.12.2
args==0.1.0
asn1crypto==1.4.0
astunparse==1.6.3
async-timeout==3.0.1
attrs==20.3.0
Automat==20.2.0
Babel==2.9.0
backports.csv==1.0.7
beets==1.4.9
bencoder.pyx==2.0.1
bidict==0.21.2
cached-property==1.5.2
cachetools==4.2.1
cchardet==2.1.7
certbot==1.13.0
certbot-nginx==1.13.0
certifi==2020.12.5
cffi==1.14.5
chardet==4.0.0
Cheetah3==3.2.6.post1
cheroot==8.5.2
CherryPy==18.6.0
click==7.1.2
clint==0.5.1
colorama==0.4.4
ConfigArgParse==1.3
configobj==5.0.6
configparser==5.0.2
constantly==15.1.0
contextlib2==0.6.0.post1
coverage==5.5
cryptg==0.2.post2
cryptography==3.4.6
cssselect==1.1.0
Cython==0.29.22
dataclasses==0.8
deemix==2.0.16
deezer-py==0.0.15
deluge==2.0.3
deprecation==2.1.0
discogs-client==2.3.0
distlib==0.3.1
distro==1.5.0
distro-info==0.0.0
dnspython==1.16.0
docopt==0.6.2
docutils==0.16
dottorrent==1.10.1
dottorrent-cli==1.10.3
eventlet==0.30.2
eyed3==0.9.6
feedparser==6.0.2
filelock==3.0.12
filetype==1.0.7
Flask==1.1.2
Flask-SocketIO==5.0.1
flatbuffers==1.12
future==0.18.2
gast==0.4.0
gdown==3.12.2
GeoIP==1.3.2
google-auth==1.27.1
google-auth-oauthlib==0.4.3
google-pasta==0.2.0
googledrivedownloader==0.4
greenlet==1.0.0
grpcio==1.36.1
h5py==3.1.0
hachoir==3.1.2
heybrochecklog @ file:///data/ash/appz/hbcl
httplib2==0.19.0
humanfriendly==9.1
hyperlink==21.0.0
idna==2.10
idna-ssl==1.1.0
imagesize==1.2.0
imgurup==1.7.0
importlib-metadata==3.7.0
importlib-resources==5.1.2
incremental==21.3.0
internetarchive==1.9.9
iotop==0.6
itsdangerous==1.1.0
jaraco.classes==3.2.1
jaraco.collections==3.2.0
jaraco.functools==3.2.1
jaraco.text==3.5.0
jeepney==0.6.0
jellyfish==0.8.2
Jinja2==2.11.3
josepy==1.7.0
jsonpatch==1.31
jsonpointer==2.0
Keras-Preprocessing==1.1.2
keyring==23.0.0
keyrings.alt==4.0.2
llvmlite==0.35.0
lossycop @ file:///data/ash/appz/lossycop-master.zip
lxml==4.6.2
Mako==1.1.4
Markdown==3.3.4
MarkupSafe==1.1.1
mediafile==0.6.0
meson==0.57.1
minidb==2.0.4
mock==4.0.3
monotonic==1.5
more-itertools==8.7.0
multidict==5.1.0
munkres==1.1.4
musicbrainzngs==0.7.1
mutagen==1.45.1
ndg-httpsclient==0.5.1
netifaces==0.10.9
notify2==0.3.1
numba==0.52.0
numpy==1.19.5
nzbverify==0.3.0
oauthlib==3.1.0
olefile==0.46
opt-einsum==3.3.0
packaging==20.9
parsedatetime==2.6
pbr==5.5.1
Pillow==8.1.2
portend==2.7.1
prettytable==2.1.0
protobuf==3.15.5
psycopg2==2.8.6
psycopg2-binary==2.8.6
ptpimg-uploader==0.5
pyaes==1.6.1
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycairo==1.20.0
pycodestyle==2.6.0
pycparser==2.20
pycrypto==2.6.1
pycryptodome==3.10.1
pycryptodomex==3.10.1
pydub==0.25.0
Pygments==2.8.1
PyGObject==3.38.0
PyHamcrest==2.0.2
PyICU==2.6
pyOpenSSL==20.0.1
pyparsing==2.4.7
pyperclip==1.8.2
PyQt5==5.15.3
PyQt5-Qt==5.15.2
PyQt5-sip==12.8.1
PyQtWebEngine==5.15.3
PyQtWebEngine-Qt==5.15.2
pyReScene==0.7
pyresceneauto==0.0.5
pyRFC3339==1.1
pySmartDL==1.3.4
PySocks==1.7.1
pyTelegramBotAPI==3.7.6
python-apt==1.6.5+ubuntu0.5
python-debian==0.1.39
python-engineio==4.0.0
python-libtorrent==1.2.6
python-socketio==5.0.4
pytz==2021.1
pywebview==3.4
pyxdg==0.27
PyYAML==5.4.1
ranger-fm==1.9.3
rencode==1.0.6
requests==2.25.1
requests-oauthlib==1.3.0
requests-toolbelt==0.9.1
requests-unixsocket==0.2.0
resampy==0.2.2
rsa==4.7.2
sabyenc3==4.0.2
scandir==1.10.0
scdl @ git+https://github.com/flyingrub/scdl@6dd6ef713bcaece9df9f8d8fd8f164862767738b
schema==0.7.4
scipy==1.5.4
sdist==0.0.0
SecretStorage==3.3.1
service-identity==18.1.0
setproctitle==1.2.2
sgmllib3k==1.0.0
six==1.15.0
snowballstemmer==2.1.0
SoundFile==0.10.3.post1
speedtest-cli==2.1.2
Sphinx==3.5.2
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==1.0.3
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.4
spotipy==2.17.1
systemd-python==234
telegram-upload==0.4.0
Telethon==1.20
tempora==4.0.1
tensorboard==2.4.1
tensorboard-plugin-wit==1.8.0
tensorflow==2.4.1
tensorflow-estimator==2.4.0
termcolor==1.1.0
texttable==1.6.3
tidal-dl==2021.2.20.0
tldr==1.2.0
toml==0.10.2
tqdm==4.59.0
Twisted==21.2.0
typing-extensions==3.7.4.3
unattended-upgrades==0.1
Unidecode==1.2.0
urllib3==1.26.3
urlwatch==2.22
vcsi==7.0.13
virtualenv==20.4.2
wcwidth==0.2.5
Werkzeug==1.0.1
wrapt==1.12.1
yarl==1.6.3
youtube-dl==2021.3.3
zc.lockfile==2.0
zipp==3.4.1
zope.component==4.6.2
zope.deferredimport==4.3.1
zope.deprecation==4.4.0
zope.event==4.5.0
zope.hookable==5.0.1
zope.interface==5.2.0
zope.proxy==4.3.5

### Description

The artist tag isn't recognized when uploading FLAC with telegram-upload. The files appear as "Artist Unknown" on Telegram while they're proprely tagged with vorbis comment. The Trackname tag is showing up fine.
This isn't a Telegram issue since uploading the file with Telegram's client will show up the Artist tag. 

### What I Did

```
telegram-upload file.flac
```

**Comment by awsms** [comment]
Possible to have an ETA on this @Nekmo ?

**Comment by Nekmo** [comment]
Thix feature will be available in release v0.6.0.

**Comment by Nekmo** [comment]
Tested in telegram-upload v0.5.0. The bug could not be reproduced.
![photo_2023-06-14_18-03-24](https://github.com/Nekmo/telegram-upload/assets/740106/245cde36-f8de-451c-9115-9627dc97c16d)
